ext/hal/st/stm32cube/stm32mp1xx/drivers/include/stm32mp1xx_ll_exti.h

```diff
@@ -1,6 +1,6 @@
 /**
   ******************************************************************************
-  * @file    stm32wbxx_ll_exti.h
```

Comment:
i would prefer to keep it separate. This is a fix to apply on top of the ST released(https://github.com/STMicroelectronics/STM32CubeMP1).

Reply:
Indeed this should be on top of initial commit, but as an introduction of the file, not as a fix of the file (since stm32mp1_ll_exti.h is not part of official ST release)